webui/src/lib/components/auth/layout.tsx

```diff
@@ -28,8 +28,10 @@ export const AuthLayout = () => {
         <Container fluid="xl">
             <Row className="mt-5">
                 <div>
+                    <Alert variant="warning" title="ACL Deprecation"><InfoIcon/>{" "}<b>ACLs are moving out of core lakeFS!</b>{"  "}See our <Alert.Link href={"https://lakefs.io/blog/why-moving-acls-out-of-core-lakefs/"}>announcement</Alert.Link>{" "}
```

Comment:
60% of installations are single user anyway, and this is perhaps too intimidating. I suggest:

1. put it only in the administration tab (dunno if hard to do, but nice to have - only show if `len(users) > 1`)
2. change the wording: from "our annuncement" to "the announcement"

Reply:
1. It's only in the administration tab + done
2. Done

Reply:
Sorry to be contrary.  Consider the fact that there is little reason for a user ever to go to the Admin tab.  I would expect an admin of a medium-sized lakeFS installation to manage users maybe a few times a year.[^1]  Such an admin would never see the deprecation banner.

[^1]:  Can we estimate whether this is true?

Reply:
Though I understand your point, I think the decision on where to show it is a product one.
@ozkatz, do you still think we should place it only on the administration tab scope?

Reply:
I think so. >60% of users shouldn't care about this change, no reason to instill fear. On the other hand, while I agree with @arielshaqed that the admin page isn't visited frequently, it's still probably a higher frequency than upgrading lakeFS versions. 

No perfect solution here..